content/en/blog/_posts/2025-xx-xx-ippr-goes-ga.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Kubernetes 1.35: In-Place Pod Resize Graduates to Stable"
+slug: in-place-pod-resize-ga
+draft: true
+date: XXXX-XX-XX
+author: >
+  Natasha Sarkar (Google)
+---
+
+It is my great honor to announce that more than 6 years after its initial conception,
+the **In-Place Pod Resize** feature (also known as In-Place Pod Vertical Scaling), first introduced as 
+alpha in Kubernetes v1.27, and graduated to beta in Kubernetes v1.33, is now **stable (GA)** in Kubernetes
+1.35!
+
+This graduation marks a major milestone for improving resource efficiency and flexibility for workloads 
+running on Kubernetes.
+
+## What is In-Place Pod Resize?
+
+In the past, the CPU and memory resources allocated to a container in a Pod were immutable. This meant changing
+them required deleting and recreating the entire Pod. For stateful services, batch jobs, or latency-sensitive
+workloads, this was an incredibly disruptive operation.
+
+In-Place Pod Resize makes CPU and memory requests and limits mutable, allowing you to adjust these resources
+within a running Pod, often without requiring a container restart.
+
+**Key Concept:**
+
+* **Desired Resources:** A container's `spec.containers[*].resources` field now represents the desired 
+resources. For CPU and memory, these fields are now mutable.
+* **Actual Resources:** The `status.containerStatuses[*].resources` field reflects the resources currently
+configured for a running container.
+* **Triggering a Resize:** You can request a resize by updating the desired `requests`
+and `limits` in the Pod's specification by utilizing the new `resize` subresource.
+
+## How can I start using In-Place Pod Resize?
+
+Detailed usage instructions and examples are provided in the official documentation:
+[Resize CPU and Memory Resources assigned to Containers](/docs/tasks/configure-pod-container/resize-container-resources/).
+
+## How does this help me?
+
+In-place Pod Resize is a foundational building block that unlocks seamless, multidimensional autoscaling and
```

Comment:
Maybe say "vertical autoscaling" to avoid confusion with https://docs.cloud.google.com/kubernetes-engine/docs/how-to/multidimensional-pod-autoscaling?
```suggestion
In-place Pod Resize is a foundational building block that unlocks seamless, vertical autoscaling and
```